plugins/pentaho-googledrive-vfs/core/src/main/java/org/pentaho/googledrive/lifecycle/GoogleDriveKettleLifecycleListener.java

```diff
@@ -0,0 +1,59 @@
+/*!
+* Copyright 2010 - 2017 Pentaho Corporation.  All rights reserved.
```

Comment:
Since these are whole new files, the copyright date should be simply Copyright 2017 Pentaho Corporation. This file wasn't created in 2010...